README.md

```diff
@@ -32,6 +32,7 @@ article for more details about how to contribute.
 1. Fork this repository into your account on Github
 2. Clone *your forked repository* (not our original one) to your hard drive with `git clone https://github.com/YOURUSERNAME/api_docs.git`
 3. Develop and validate your changes (see below)
+4. Update the changelog with a breif summary of the changes you made.
```

Comment:
brief

Reply:
Gah! Just merged a moment before, will update in new PR.